taylor-fatima.js

```diff
@@ -0,0 +1,143 @@
+
+//Part 1
+//#1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(num) {
+    console.log(Math.pow(num, 3))
+  }
+  calculateCube(2)
+  
+  //#2
+  //Write a function isAVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+  
+  function isAVowel(letter) {
+    if (letter === 'a') {
+      return true
+    } else if (letter === 'e'){
+      return true
+    } else if (letter === 'i') {
+      return true
+    } else if (letter === 'o') {
+      return true 
+    } else if (letter === 'u') {
+      return true
+    } else {
+      return false
+    }
+  }
+  console.log(isAVowel('a'))
+  
+  //#3
+  //Write a function getTwoLengths that accepts two parameters (strings). 
+  //The function should return an array of numbers where each number is the length of the corresponding string.
+  
+  function getTwoLengths(str1, str2) {
+   let length = [];
+   length.push(str1.length)
+   length.push(str2.length)
+   return length
+  }
+  console.log(getTwoLengths("hi", "hello"))
+  
+  //#4
+  // Write a function getMultipleLengths that accepts a single parameter as an argument: an array of strings. The function should return an array of numbers where each number is the length of the corresponding string.
+  
+  function getMultipleLengths(arr) {
+    let newArray = [];
+    for (let i = 0; i < arr.length; i++) {
+      newArray.push(arr[i].length) 
+    }
+    return newArray
+  }
```

Comment:
nice! It looks like from line 9 down your code is always indented one level more than it should be, but other than that, this function's organization, syntax, use of white space, etc. are on point 💯